config/newrelic.yml

```diff
@@ -0,0 +1,49 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python and Node applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated September 17, 2015
+# 
+# This configuration file is custom generated for SumOfUs
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: <%= ENV['NEWRELIC_LICENSE_KEY'] %>
```

Comment:
Where did this env var get added / did this env var get added to EB and CircleCI?